app/points/calculate-available-points.js

```diff
@@ -0,0 +1,37 @@
+const DefaultContractedHours = require('./domain/default-contracted-hours')
+const OM_TYPE_IDS = require('./constants/offender-manager-type-ids')
+
+var calculateAvailablePoints = function (nominalTarget, baseHours, hoursReduction, defaultContractedHoursForBand) {
+  return (nominalTarget * (baseHours / defaultContractedHoursForBand)) * ((baseHours - hoursReduction) / baseHours)
+}
+
+module.exports = function (nominalTarget, offenderManagerTypeId, contractedHoursPerWeek,
+  hoursReduction, defaultContractedHours) {
+  if (!(defaultContractedHours instanceof DefaultContractedHours)) {
+    throw new Error('defaultContractedHours should be an instance of DefaultContractedHours')
+  }
+
+  // TODO: could maybe store this as part of the dict with the values as a hasAvailability flag?
```

Comment:
As discussed can you port this over to the backlog as a chore detailing the information we would need in place before implementing it and remove the comment? Cheers.